packages/react-percy-config/src/getWebpackSettings.js

```diff
@@ -0,0 +1,46 @@
+import path from 'path';
+
+function sanitizeCustomWebpackConfig(config = {}) {
+  // Do not allow users to customize the entry config
+  delete config.entry;
+
+  // Do not allow users to customize the output config
+  delete config.output;
+
+  // Move module.loaders to module.rules so we can properly merge with them
+  if (config.module && config.module.loaders) {
+    config.module.rules = config.module.loaders;
```

Comment:
Just to be safe, if a user sets `webpack.module.loaders` in `percy.config.js`, we'll automatically move it over to `webpack.module.rules` so it works as expected.